bin/build-tasks-doc.js

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const path = require('path');
+const got = require('got');
+const dedent = require('dedent');
+
+const npmUrl = 'https://registry.npmjs.com/'
+const taskList = require('../tasks.json');
+const tasksHeaderPath = path.join(__dirname, 'tasks-header.md');
+const tasksHeader = fs.readFileSync(tasksHeaderPath, 'utf8');
+const tasksOutputFilePath = path.join(__dirname, '..', 'docs', 'tasks.md');
+
+function catchError (err) {
+  console.log(err);
+  process.exit(1);
+}
+
+function getTaskPkg (taskName) {
+  // npm registry is weird. it wants slashes to be uri encoded but not @ symbols
+  const url = npmUrl + taskName.split('/').join('%2F');
+  return got(url, { json: true }).then((res) => res.body);
+}
+
+function createTaskResourceLinks (packageName, sourceUrl, homepage) {
+  let resources = `[npm](https://npmjs.com/packages/${packageName}) `
```

Comment:
The npm links do not work. I think it should be `package`, not `packages`, i.e. https://www.npmjs.com/package/@cumulus/parse-pdr